specs/Terminate Spec.md

```diff
@@ -0,0 +1,60 @@
+# **Terminate Program Spec**
+
+![MS PowerToys](https://hothardware.com/ContentImages/NewsItem/48038/content/Microsoft_PowerToys.jpg "PowerToys")
+## Shortcut for users to quickly terminate a running program
+### Authors: Benjamin Leverette and Prudence Phillips
+### Spec Status: Draft
+## 1. Overview
+
+**1.1. Elevator Pitch / Narrative** 
+
+Mike is debugging his code in Visual Studio and the program freezes with the "App not responding" text displaying on the title bar, informing him that he cannot continue his work without closing the program and restarting it. Mike can completely shutdown the app quicker using a Keyboard shortcut if he has this PowerToy installed on his computer.
+
+**1.2. Customers**
+
+PowerToys is mainly targeted towards Windows Power Users though it is available to users who want to experience using windows in a more efficient and productive way.
+ 
+  
+**1.3. Problem Statement and Supporting Customer Insights**
+
+Windows users need a quicker, more efficient way to completely shut down an app whenever it is unresponsive. The team is still required to find a proper name for this PowerToy.
+
+**1.4. Existing Solutions or Expectations**
+
+Users can already acheive the function of this PowerToy by accessing clicking End Task through Task Manager on Windows. Users need to be running Windows 10 and have PowerToys installed to use it.
+
+
+**1.5. Goals/Non-Goals**
+
+- Develop this PowerToy and have sufficient time for testing and integration within our assigned 8 weeks for the project. 
+
+- Come up with a suitable name for this PowerToy, preferrably one accepted by the community.
+
+## 2. Definition of Success
+
+**2.1. Expected Impact: Customer, and Technology Outcomes, Experiments + Measures**
+
+There is currently a sizable number of people who are following the PowerToys project. We hope to meet these people's expectations and receive good feedback from the community when we are done with this PowerToy.
+
+## 3. Requirements
+
+**3.1.	Functional Requirements**
+
+- A keyboard shortcut to display a confirmation window 
+- Cursor morphs when user hovers over the "Yes" option
+- Normal cursor when user hovers over the "No" option 
+
+![Terminate](https://raw.github.com/indierawk2k2/PowerToys-1/images/PT%20Images/Terminate%20Blurred.png "Terminate")
+
+- Keyboard shortcut suggestions
+    - Windows Key + Alt + X
+    - Windows Key + Shift + X 
+
+**3.2. Measure Requirements**
+
+Communicating with the community through the means of surveys and Github issues.
```

Comment:
Measures usually need to define what they treat as success with a numeric value or equation. "95% NSAT score in survey", "response rate of 10% of the repository star count", etc.